topi/python/topi/nn/convolution.py

```diff
@@ -106,98 +107,194 @@ def conv2d_hwcn(Input, Filter, stride, padding):
         name="Conv2dOutput", tag="conv2d_hwcn")
     return Output
 
-def depthwise_conv2d_nchw(Input, Filter, stride, padding):
-    """Depthwise convolution nchw forward operator.
 
-    Parameters
-    ----------
-    Input : tvm.Tensor
-        4-D with shape [batch, in_channel, in_height, in_width]
+workload_entity = ['height', 'width', 'in_filter', 'out_filter',
```

Comment:
any idea where to put those?

Reply:
I think we can define them here, as long as the naming convention is good

Reply:
there is another duplicated definition in schedule, so it is better to keep it at another common place